hadoop-ozone/dist/src/main/compose/xcompat/clients.yaml

```diff
@@ -17,15 +17,6 @@
 version: "3.4"
 
 services:
-  old_client_0_5_0:
```

Comment:
Can you provide some context on why this has been removed?

Reply:
From PR description:

> 0.5.0 client had to be removed from the test due to another incompatibility:
> 
> ```
> Message missing required fields: getFileStatusResponse.status.status, ...
> ```